### DISCUSSION TITLE
unable to run ipxe.efi in uefi shell

### DISCUSSION THREAD

**Discussion by badiku** [ipxe/ipxe#736] [General]
hi, how to run ipxe.efi in uefi shell?

FS0:
cd /efi/boot
ipxe.efi

Command Error Status: Unsupported

![pxe](https://user-images.githubusercontent.com/30396109/188809853-0fbfa815-deac-4287-ab9b-f451c424bdc9.jpg)

use ifconfig -s eth0 dchp
ifconfig -l eth0
ping 192.168.1.5 
I can see that it can obtain ip from DHCP server and ping server properly.

so ipxe.efi can only be downloaded by tftp?

I know that I can load ipxe from grub or from iso, but why cannot run the ipxe.efi directly in shell?

**Comment by NiKiZe** [comment]
That normally works, but be sure that you are using a valid binary, and check secure boot?

**Reply by badiku** [reply]
ok my fault actually  I use wrong file

**Reply by badiku** [reply]
seems that symantec thought there's virus in ipxe.efi and do sth to repair it. strange.
